package.json

```diff
@@ -60,7 +62,6 @@
     "@typescript-eslint/parser": "^5.3.1",
     "babel-loader": "^8.2.5",
     "cypress": "^9.7.0",
-    "downlevel-dts": "^0.10.0",
```

Comment:
Just for sanity/clarity - we will need to add this back in a later PR for the 4.13 OCP console dynamic plugins epic: https://issues.redhat.com/browse/CONSOLE-3304

Reply:
Also @vojtechszocs we may want to consider splitting up unrelated changes in the future. It will reduce the diff (already very large) and make it faster for approval.